RxCocoa/iOS/UISearchBar+Rx.swift

```diff
@@ -47,6 +47,27 @@ extension UISearchBar {
         
         return ControlProperty(values: source, valueSink: bindingObserver)
     }
+    
+    /**
+    Reactive wrapper for `scope` property.
+    */
+    public var rx_scope: ControlProperty<Int> {
```

Comment:
I think for consistency sake, we should name this property, `rx_selectedScopeButtonIndex`.